src/dynamo/request/query/query.request.ts

```diff
@@ -132,4 +133,8 @@ export class QueryRequest<T> extends Request<T, QueryRequest<T>, QueryInput, Que
       })
     )
   }
+
```

Comment:
add some doc, side note about how this can eat all the provisioned output and to rather use pagedDatasource for client side use cases